meetings/2022-06-30.md

```diff
@@ -39,38 +42,99 @@
-
-
+* SC: This adresses some of the considerations wrt #58, how much we rely on the Solid Protocol and how much the Notifications Protocol covers.
+* SC: There might be related comments in #76.
+* SC: Discovering the features that are available gives a good idea of what is missing. It doesn't matter where that place for discovery is, but now we have a reliable place where this information lives with the `solid:storageDescription`.
+* SC: If the Notifications Protocol relies on that feature, then it can say what the channel/subscription information should be available there.
+* CB: Maybe a naive question: Why do we not rely on the .meta resource? 
+* SC: I think the URL for that is implementation specific - was it NSS? So we need a FYN type of approach, typically with a link relation to find that.
+* CB: Isn't that also in the Link-Header? 
+* SC: Oh, I think you are describing the `describedby` relation which some implementations use `.meta` as the resource. That's related. That relation can be used on any resource about itself / quote, unquote metadata, but the storageDescription is only about the storage description in which the storage that the resource is in.
+* CB: I see, I thought that in such a meta resource, one could announce that for this resource such notifications would be possible, but if notifications are tied storage, then this makes more sense.
+* SC: Yea, that's part of the considerations in #58 e.g., endpoint per storage for all resources or unique endpoint per resource. Different ways to go at it.. we need to understand/weigh it out.
+* SC: I don't think it makes a whole lot difference what the design is for clients. But matters for servers.
+* CB: right.
+* RG: Has there been any discussion on which way you want to go?
+* SC: I think we somehow defaulted to storage because that seemed "good enough" coverage. It would make the wiring of that information per resource more complicated, whether that's done by out of band by the server/owner or some specific mechanism that allows clients ot do it. It gets complicated for sure ot assign a different channel description info per resource.
+* SC: How important is the use cases to assign different channels for different resources in the same storages (or even in the same container).
+* RG: one client might have access to foo an dbar. another to foo but not bar. the second client connects, tries to access bar, that access should be foribidden in some way.
+* CB: I imagine that's handled by the notification server/channel service the usual with the HTTP request would be handled by the authz.
+* RG: What happens with bar inside foo?
+* CB: I am a client that has access to foo/ but not bar, then i'd be able to subscribe to foo/ and then the notification channel service accepts my request because i have access to foo/ and then if i want to subscribe to foo/bar then server will reject that because im not authorized. i'm not sure how this is related to how the channel is described.
```

Comment:
```suggestion
* CB: If I am a client that has access to `foo/` but not `bar`, then i'd be able to subscribe to `foo/`, and the notification channel service accepts my request because I have access to `foo/`. Then if I want to subscribe to `foo/bar`, the server will reject that, because I'm not authorized. I'm not sure how this is related to how the channel is described.
```